### PR TITLE
refactor encryption service for checksum

### PR DESCRIPTION
**Pull request by sandeep-groww**
## What does this PR do?
Supports cipher option as a param in encrypt function . Will be used by checksum service

## Types of changes

What types of changes does your code introduce to this project?

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)

- [X] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Package version increase in any of the packages?
encryption 1.0.0 ==> 1.0.1

## Checklist before merging

_Put an `x` in the boxes that apply_

- [ ] These changes have been thoroughly tested.

- [ ] Changes need to be immediately published on npm.

### CODE REVIEW

File: packages/encryption/src/types.ts [hunk 1]
```diff
@@ -2,3 +2,9 @@ export type ResponseType = {
   data: string | null;
   error: string | null;
 }
+
+export type CipherOptionType = {
+  iv?: CryptoJS.lib.WordArray | undefined;
```
Comment (shubhamgavali1998) [inline packages/encryption/src/types.ts:7]: No need to write undefined if it is optional type

File: packages/encryption/src/types.ts [hunk 2]
```diff
@@ -2,3 +2,9 @@ export type ResponseType = {
...
+  format?: any ;
```
Comment (shubhamgavali1998) [inline packages/encryption/src/types.ts:8]: Can we avoid any and have a proper type?

File: packages/encryption/src/index.ts [hunk 3]
```diff
@@ -45,6 +50,51 @@ export const encryptAes = (dataToEncrypt: object | string, ENCODE: string) => {
...
+ * ```
+ * encryptAes('Data to encrypt','secret-key')); => Output will be
+ * {
+ *  data:'##Some random ciphered text',
+ *  error:null
+ * }.
+ * ```
+ */
+
+export const encryptAesBase64 = (dataToEncrypt:string, ENCODE: string| CryptoJS.lib.WordArray, cipherOptions:CipherOptionType) => {
```
Comment (shubhamgavali1998) [inline packages/encryption/src/index.ts:76]: Format this code

File: packages/encryption/src/types.ts (hunk 2 above)
Comment (sandeep-groww) [reply packages/encryption/src/types.ts:8]: The library itself uses any . So cannot be fixed

**Pull request merged**